libflux/src/flux/semantic/nodes.rs

```diff
@@ -1545,13 +1544,23 @@ impl DateTimeLit {
     }
 }
 
+// DurationLit is a pair consisting of length of time and the unit of time measured.
+// It is the atomic unit from which all duration literals are composed.
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+#[serde(rename = "Duration")]
+pub struct Duration {
+    pub months: i64,
```

Comment:
we should comment that these fields should always contain positive values

Reply:
Yes, I'll add a comment here. I also need to be taking the absolute value of the final nanoseconds and months values in `convert_duration()`. I'll update that.